src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/InstallAppsTask.groovy

```diff
@@ -141,12 +141,34 @@ class InstallAppsTask extends AbstractServerTask {
 
 
     String getArchiveOutputPath() {
-        if (springBootVersion.startsWith('2')) {
-            return project.bootJar.archivePath.getAbsolutePath()
+        String archiveOutputPath;
+
+        if (springBootVersion.startsWith('2.')) {
+            archiveOutputPath = project.bootJar.archivePath.getAbsolutePath()
```

Comment:
Do we need to check `bootWar' here too? Just wondering since we're checking the war case for 1.5.

Reply:
Also, since we're checking the archiveOutputPath below, do we even want to check the war case? Not sure how SpringUtil.isSpringUberJar() works with the war case or if it is something we're supporting.

Reply:
Done. Added bootWar check